week-4/implement/is-prime.test.js

```diff
@@ -1,3 +1,26 @@
 // Given a positive integer num,
 // When the isPrime function is called with num as input,
 // Then it should return a boolean representing whether the num is prime
+
+function isPrime(num) {
+    if (num < 2) {
+        return false;
+    }
+    if (num != Math.round(num)) {
+        return false;
+    }
+    for (var i = 2; i < num; i++){
```

Comment:
Great spot that you can begin this loop at 2! I would just replace `var` with `let`, as this helps to keep the variable scope as small as possible, in this case, only inside the `for` loop. Otherwise you would be able to access `i` outside of the loop which could cause bugs.